src/pkg/cluster/injector.go

```diff
@@ -28,6 +28,13 @@ import (
 // The chunk size for the tarball chunks.
 var payloadChunkSize = 1024 * 768
 
+var (
+	requestedCPU    = resource.MustParse(".5")
+	requestedMemory = resource.MustParse("64Mi")
+	limitCPU        = resource.MustParse("1")
+	limitMemory     = resource.MustParse("256Mi")
```

Comment:
tiny thing on these - since they are specific to the injector pod they should include a reference in the name incase people see them elsewhere when typing `cluster.` with IntelliSense

```suggestion
	injectorRequestedCPU    = resource.MustParse(".5")
	injectorRequestedMemory = resource.MustParse("64Mi")
	injectorLimitCPU        = resource.MustParse("1")
	injectorLimitMemory     = resource.MustParse("256Mi")
```